ansible/roles/kong-api/defaults/main.yml

```diff
@@ -4855,4 +4855,58 @@ kong_apis:
      - name: request-size-limiting
        config.allowed_payload_size: "{{ small_request_size_limit }}"
 
+  - name: createTenantPreferences
+    uris: "{{ org_service_prefix }}/v2/preferences/create"
+    upstream_url: "{{ learning_service_url }}/v2/org/preferences/create"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'orgPreferenceCreate'
```

Comment:
We don't need any new ACLs for preference @aishwa8141 , we can reuse the existing ACLs

Reply:
done